src/cbc_sdk/platform/alerts.py

```diff
@@ -203,6 +206,83 @@ def _query_implementation(cls, cb, **kwargs):
         """
         return CBAnalyticsAlertSearchQuery(cls, cb)
 
+    def get_events(self, timeout=0, async_mode=False):
+        """Requests enriched events detailed results.
+
+        Args:
+            timeout (int): Event details request timeout in milliseconds.
+            async_mode (bool): True to request details in an asynchronous manner.
+
+        Returns:
+            list: EnrichedEvents matching the legacy_alert_id
+
+        Note:
+            - When using asynchronous mode, this method returns a python future.
+              You can call result() on the future object to wait for completion and get the results.
+        """
+        self._details_timeout = timeout
+        alert_id = self._info.get('legacy_alert_id')
```

Comment:
This might change depending on the conversation on Tuesday.

Reply:
Note if it is a Watchlist Alert you need to use `id` but for CB_ANALYTIC alerts you need to use `legacy_alert_id`. I'm not sure what it would be for a Device Control alert

Reply:
Only Analytics alerts seem to be returning Enriched events for the moment.